libs/api/domains/license-service/src/lib/licenseService.service.ts

```diff
@@ -456,22 +484,191 @@ export class LicenseServiceService {
       )
     }
 
-    if (!licenseService.verifyPkPass) {
-      this.logger.error('License client has no verifyPkPass implementation', {
-        category: LOG_CATEGORY,
+    if (
+      !licenseService.verifyPkPassDeprecated ||
+      !licenseService.verifyPkPass
+    ) {
+      const missingMethodMsg =
+        'License client has no verifyPkPass nor verifyPkPassDeprecated implementation'
+      this.logError(missingMethodMsg, {
         passTemplateId,
       })
+
       throw new BadRequestException(
-        `License client has no verifyPkPass implementation, passTemplateId: ${passTemplateId}`,
+        `${missingMethodMsg}, passTemplateId: ${passTemplateId}`,
       )
     }
 
-    const verification = await licenseService.verifyPkPass(data, passTemplateId)
+    if (licenseService?.verifyPkPass) {
```

Comment:
Should first try `verifyPkPassDeprecated` if it exists. Otherwise this will return the `picture` field instead of `photo` like old clients expect.